src/test/java/seedu/address/model/cheese/CheeseTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.cheese;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CHEESE_TYPE_CAMEMBERT;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_CHEESE_TYPE_FETA;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_1;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EXPIRY_DATE_2;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MANUFACTURE_DATE_1;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_MANUFACTURE_DATE_2;
+import static seedu.address.testutil.Assert.assertThrows;
+import static seedu.address.testutil.TypicalCheese.CAMEMBERT;
+import static seedu.address.testutil.TypicalCheese.FETA;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.CheeseBuilder;
+
+/**
+ * Represents a Cheese in the address book.
+ */
+public class CheeseTest {
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new CheeseBuilder().withCheeseType(null).build());
+    }
+
+    @Test
+    public void isSameCheese() {
+        // same object -> return true
```

Comment:
Standardize all comments to start with capital letter?

Reply:
Would be nice to standardize but might not be necessary (will have a lot of existing codes to update)

Will bring it up in meeting tmr